modules/obtaining-ocp-packages-restricted-network-cluster.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc
+
+[id="obtaining-ocp-packages-restricted-network-cluster_{context}"]
+= Obtaining {product-title} packages
+
+Before you can update your {op-system-base-full} compute machines on a restricted network cluster, you must obtain the required images and components and store them in your repository.
+
+[IMPORTANT]
+====
+You must obtain the required images and software components on a system with the same architecture as the cluster that is in your disconnected environment.
+====
+
+.Prerequisites
+
+* Obtain a {op-system-base-full} 7 server that you have root access to, with access to the internet and at least 110 GB of disk space. Download the required software repositories and container images to your computer.
```

Comment:
Which version of OCP will this PR be applied to? If I remember right, different version of OCP support different version of RHEL (7 or 8).